plugins/woocommerce/includes/class-wc-cart-fees.php

```diff
@@ -51,16 +43,18 @@ final class WC_Cart_Fees {
 	/**
 	 * Constructor. Reference to the cart.
 	 *
+	 * @param null $deprecated Deprecated since WooCommerce 8.0.4.
```

Comment:
This will need to be updated to `8.2.0`. For reference for future PRs, bug fixes are released in `X.X.0` unless they are fixing a bug that was introduced in a `.0` release.